src/Guzzle6HttpAdapter.php

```diff
@@ -8,15 +8,15 @@
  * For the full copyright and license information, please read the LICENSE
  * file that was distributed with this source code.
  */
-
 namespace Http\Adapter;
 
 use GuzzleHttp\Client;
 use GuzzleHttp\ClientInterface;
-use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception as Guzzle;
```

Comment:
It is actually 6. The reson behind this is to be more clear on which exceptions that are from Guzzle and which are not. 

Sure, will do.